app/views/posts/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<a href="<%= user_path(@user) %>">
+  <button id="back-btn">⬅ Back to User Profile</button>
+</a>
+<h1>Create a New Post</h1>
+
+<%= form_with model: [@user, @post], local: true do |form| %>
+  <% if @post.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@post.errors.count, "error") %> prohibited this post from being saved:</h2>
+      <ul>
+        <% @post.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= form.label :title %>
```

Comment:
- [ ] I'm able to create post, but I have to go to this page manually, Please include button in post's index page so that navigation is easier.